rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -16,12 +15,21 @@
 
 @pytest.fixture(name='tokens')
 def fixture_ethereumtokens(ethereum_inquirer, database, inquirer):  # pylint: disable=unused-argument  # noqa: E501
-    return EthereumTokens(database, ethereum_inquirer)
+    return EthereumTokens(database, ethereum_inquirer, proxies_mapping={})
 
 
-@flaky(max_runs=3, min_passes=1)  # failed in a flaky way sometimes in the CI due to etherscan
+# @flaky(max_runs=3, min_passes=1)  # failed in a flaky way sometimes in the CI due to etherscan
```

Comment:
why did you comment this out? IF you want to remove it, remove it.

But you would have to make sure **all** networking is properly mocked and this passes by using the pytest socket plugin

Reply:
Forgot to uncomment after testing locally, thanks for noticing